### DISCUSSION TITLE
Issue with loading AnvilWorld if the world has negative -80 coords

### DISCUSSION THREAD

**Discussion by KrunkerPremium** [Minestom/Minestom#1907] [Help] [answered]
getExtension().lobbyManager = new LobbyManager(getExtension(),MinecraftServer.getInstanceManager().createInstanceContainer(new AnvilLoader("world")));

this is the line in my code where I try to load the world using the default Overworld type with the minheight set to -50 but I get an error saying it needs to be -80 because of my world chunk having a -80

**Comment by KrystilizeNevaDies** [comment, marked as answer]
You will need to create an instance with a `DimensionType` that has your world's specific min/max height.
e.g.
```java
DimensionType dimension = DimensionType.builder("minestom:example_dimension")
  .minY(-80)
  .maxY(1000)
  .build();
MinecraftServer.getDimensionManager().register(dimension);
Instance instance = MinecraftServer.getInstanceManager().createInstanceContainer(dimension);
```
